tests/regression/test_helmholtz.py

```diff
@@ -77,6 +77,19 @@ def test_firedrake_helmholtz_scalar_convergence_on_quadrilaterals(testcase, conv
     assert (np.array([np.log2(l2err[i]/l2err[i+1]) for i in range(len(l2err)-1)]) > convrate).all()
 
 
+@pytest.mark.parametrize(('testcase', 'convrate'),
+                         [((1, (4, 6)), 1.9),
+                          ((2, (3, 6)), 2.9),
+                          ((3, (2, 4)), 3.9),
+                          ((4, (2, 4)), 4.7)])
+def test_firedrake_helmholtz_scalar_convergence_on_quadrilaterals_dpc(testcase, convrate):
```

Comment:
There is already a separate file that tests the serendipity element. I don't think we need this (another note: the name is incorrect. this is not a test for dPc)